3.1.7 Release Notes

daschl opened the pull request.

daschl committed:
3.1.7 Release Notes

maria-robobug commented on modules/project-docs/pages/sdk-release-notes.adoc:
```diff
@@ -109,6 +109,48 @@ Add support for SDK2-compatible `LegacyTranscoder`.
+Ignore null values for attributed in OpenTelemetry and OpenTracing.
+* https://issues.couchbase.com/browse/JVMCBC-990[JVMCBC-990]:
+Gracefully handle cluster-level WaitUntilReady against clusters < 6.5.
+* https://issues.couchbase.com/browse/JCBC-1850[JCBC-1850]:
```
Think this is a duplicate of [line 135](https://github.com/couchbase/docs-sdk-java/pull/259/files#diff-379fc9c9c5051ce7ba1f76ca5755d42e5d8b4290cda4c25840e5d8aaa117d01eR135)

maria-robobug committed:
Gardening

- Removes duplicate note for https://issues.couchbase.com/browse/JCBC-1850
- Slight re-wording for https://issues.couchbase.com/browse/JVMCBC-988

maria-robobug commented:
👍 LGTM
